src/components/PreviewFrame.jsx

```diff
@@ -29,7 +29,7 @@ class PreviewFrame extends React.Component {
 
     if (this._channel && isActive) {
       for (const [key, {expression}] of newProps.consoleEntries) {
-        if (!previousConsoleEntries.has(key)) {
+        if (!previousConsoleEntries.has(key) && expression) {
```

Comment:
I used the existing Record for `ConsoleEntry` and `ConsoleEntry` component, with an empty expression and a check here to not evaluate expressions that are falsy. 

Wanted your opinion - this seemed a little simpler, but I could also see us using different Record and different component for console log entries - we'd have to add a 'type' to the console state to know which one to render, but wouldn't be too bad.

Reply:
ConsoleEntry now has a default null expression and ConsoleEntrys created by logs just use that default.